boards/px4/fmu-v6x/default.px4board

```diff
@@ -6,6 +6,7 @@ CONFIG_BOARD_SERIAL_GPS2="/dev/ttyS7"
 CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS6"
 CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS4"
 CONFIG_BOARD_SERIAL_TEL3="/dev/ttyS1"
+CONFIG_BOARD_SERIAL_EXT2="/dev/ttyS3"
```

Comment:
This needs to be added to:
- https://github.com/PX4/PX4-Autopilot/blob/main/cmake/kconfig.cmake#L306
- https://github.com/PX4/PX4-Autopilot/blob/main/Tools/serial/generate_config.py#L144

Reply:
@davids5 why did you want it to be called ETX2? I think it would be much more consistent to keep it PPB as this is how the port will be labeled and it is consistent with the code for V5X:
[](https://github.com/PX4/PX4-Autopilot/blob/a5e4295029162cbc66c3e61f7b11a9672a461bc4/boards/px4/fmu-v5x/default.px4board#L9)

Reply:
Because this repo is the code for the VxX_FMUM_RCxx_PINREF and VxX_BASE-T_RCxx_PINREF designs which is documented in the DS series specs. The naming is tied to the schematics and connector names not a specific companies usages or base board.

Reply:
CONFIG_BOARD_SERIAL_PPB="/dev/ttyS3"  should be changed too